### PR TITLE
feat: Adding DSC to WebTraining Day4 (PTL-1001)

### PR DESCRIPTION
**Pull request by dbarros1979**
Thank you for contributing to the documentation.

Do the changes you have made apply to both Current and Previous versions?
Yes

Have you checked all new or changed links?
Yes

Is there anything else you would like us to know?
No

For reference: 

  - We have an [internal contributions guide](https://www.notion.so/genesisglobal/Contributing-new-documentation-75953fb245f246ff872789035451a0c4)
  - We have a [style guide](https://www.notion.so/genesisglobal/Documentation-style-guide-5b04ec6fe12f4262b90d192effd8059b) 

**This week's exciting excerpts from the style guide**

- We write in UK English, with Oxford English Dictionary (OED) spellings, grammar and vocabulary.  

Use the present tense wherever possible. Only use another tense where it is strictly necessary.

- Present tense (preferred whenever possible): *The script creates a new folder.*
- Future tense (avoid whenever possible): *The script will create a new folder.*

### CODE REVIEW

**Comment by dbarros1979** [comment]
@GuiWar , @mahamzafar990 need your review/approval on this please.

**Comment by GuiWar** [comment]
Link to the DSC is not working!

File: docs/01_getting-started/10_web-training/04_web-training-day4.md [hunk 1]
```diff
@@ -923,6 +923,35 @@ You can register your own design system to make your app(s) look and feel cohesi
...
+    src={require("@site/static/img/DSC-checkbox.PNG").default}
+    alt="Image capture of DSC app"
+  />
+  <div>
+    <a class="overlay-link" href="/dsc/" target="_blank">
+      Launch DSC &#x2197;
+    </a>
+  </div>
+</div>
+<br/>
```
Comment (GuiWar) [inline docs/01_getting-started/10_web-training/04_web-training-day4.md:944]: Link to DSC not working

File: docs/01_getting-started/10_web-training/04_web-training-day4.md [hunk 2]
```diff
@@ -923,6 +923,35 @@ You can register your own design system to make your app(s) look and feel cohesi
...
+
+Experiment with adjusting the configurations and observe the corresponding changes on the right side. To use the Web Developer Training starting repository you cloned from [here](https://github.com/genesiscommunitysuccess/webtraining-seed):
+
+1. Launch the DSC.
+2. Customise the look and feel, then click on **Save Changes**.
+3. Copy the text from the pop-up CSS, and replace the content of the [design-tokens.json](https://github.com/genesiscommunitysuccess/webtraining-seed/tree/main/client/web/src/styles/design-tokens.json) file.
```
Comment (GuiWar) [inline docs/01_getting-started/10_web-training/04_web-training-day4.md:950]: 404 link not found

**Review by GuiWar — CHANGES_REQUESTED** [review]
some of the links are not working.

**Review by GuiWar — APPROVED** [review]

**Pull request merged**